lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -30,17 +43,19 @@
     cursor: pointer;
 
     .nx-radio-checkbox__control {
-      border-color: var(--nx-color-interactive-border-hover);
+      @include inset-box-shadow-border(var(--nx-color-interactive-border-hover));
```

Comment:
Why is this being done with box-shadow?

Reply:
> Why is this being done with box-shadow?

To fix it offsetting the checkmark icon, since it is now 2px in dark mode instead of 1

Reply:
> Why is this being done with box-shadow?

If it's about positioning the checkmark, consider that the border isn't visible when the checkmark is there, and really could be removed entirely in that state

Reply:
> > Why is this being done with box-shadow?
> 
> If it's about positioning the checkmark, consider that the border isn't visible when the checkmark is there, and really could be removed entirely in that state

Hmm, that would also complicate the focus styling because it also offsets the calculation for that .. i think